src/keria/app/cli/commands/migrate.py

```diff
@@ -0,0 +1,56 @@
+# -*- encoding: utf-8 -*-
+"""
+KERIA
+keria.cli.keria.commands module
+
+"""
+import argparse
+
+from hio.base import doing
+from keri.app.cli.commands.migrate import run
+from keria.db import basing
+
+
+def handler(args):
+    """
+    Migrate KERIA agents
+
+    Args:
+        args(Namespace): arguments object from command line
+    """
+    migrator = MigrateDoDoer(args)
+    return [migrator]
+
+
+parser = argparse.ArgumentParser(description='Runs outstanding migrations for all agents')
+parser.set_defaults(handler=handler,
+                    transferable=True)
+parser.add_argument('--base', '-b', help='additional optional prefix to file location of KERI keystore',
+                    required=False, default="")
+
+
+class MigrateDoDoer(doing.DoDoer):
+    def __init__(self, args):
+        self.args = args
+        self.adb = basing.AgencyBaser(name="TheAgency", base=args.base, reopen=True, temp=False)
+        doers = [doing.doify(self.migrateAgentsDo)]
```

Comment:
Let's get away from using `doing.doify` since Sam has cautioned against this. I am willing to show how to do this by refactoring the PR. You replace `doing.doify` with a Doer class that has the logic called from the `Doer.recur()` function.

I will write a HIO and KERIpy style guide to help communicate some of these ideas.